### PR TITLE
Add missing label, fieldset, and JS alt/text fixes

### PR DESCRIPTION
**Pull request by dracos**
Fixes https://github.com/mysociety/societyworks/issues/3437.
Fixes https://github.com/mysociety/societyworks/issues/3434.
Fixes https://github.com/mysociety/societyworks/issues/3431.
Part of https://github.com/mysociety/societyworks/issues/3429.

### CODE REVIEW

**Comment by dracos** [comment]
@lucascumsille Not doing anything wrong. I was fixing the actual issue raised but you're right, we should do better than that while we're here. I think it should focus on the close button when opened (so you can get out if that's not what you want, and tab would take you to next image link), and focus back where you were when it closes. I'll take a look at that.

**Comment by dracos** [comment]
As well as the additional text, I've made some focus changes: make close focusable, show next/prev on focus, focus on close button on opening, and focus back to the associated thumbnail on closure.

**Comment by lucascumsille** [comment]
> As well as the additional text, I've made some focus changes: make close focusable, show next/prev on focus, focus on close button on opening, and focus back to the associated thumbnail on closure.

@dracos I just tested it, it's working for me much easier to navigate thanks.

File: templates/web/base/alert/_list.html [hunk 1]
```diff
@@ -6,7 +6,10 @@
   <input type="hidden" name="latitude" value="[% latitude | html %]">
   <input type="hidden" name="longitude" value="[% longitude | html %]">
 
+<fieldset>
+ <legend>
   <h2>[% loc('Which problems do you want alerts about?') %]</h2>
+ </legend>
```
[the remark this answers is not in the document]
Comment (dracos) [reply templates/web/base/alert/_list.html:12]: It does read the legend with the h2 here, on the first entry in the fieldset (the RSS link in this case). I've improved the source order to have the radio/inputs first and then the RSS feed links (first RSS feed displays a bit lower but I think that's fine).

**Pull request merged**